<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Food_api</class>
 <widget class="QMainWindow" name="Food_api">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1523</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="get_recipe">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>80</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>get recipe</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>941</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>941</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Input ingredients:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>941</width>
      <height>471</height>
     </rect>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_db">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>80</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>clear local database</string>
    </property>
   </widget>
   <widget class="QComboBox" name="local_recipes">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>561</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QComboBox::AdjustToMinimumContentsLength</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="get_random_recipe">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>80</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>get random recipe</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_from_db">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>80</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>delete from local database</string>
    </property>
   </widget>
   <widget class="QLabel" name="image_area">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>120</y>
      <width>556</width>
      <height>370</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>textBrowser</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>clear_db</tabstop>
  <tabstop>local_recipes</tabstop>
  <tabstop>get_random_recipe</tabstop>
  <tabstop>get_recipe</tabstop>
  <tabstop>delete_from_db</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
